pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[tool.black]
+line-length = 88
```

Comment:
If this needs to be specified, why not stick to PEP 8?

Reply:
88 is the default line length for black. I'm mentioning 88 here to make the line length setting explicit.